### DISCUSSION TITLE
[Bug]: Possible error in the example examples/kafka/kafka-with-node-pools.yaml

### DISCUSSION THREAD

**Discussion by ignaciogonzalezsomoza** [strimzi/strimzi-kafka-operator#9915] [General]
### Bug Description

In the example examples/kafka/kafka-with-node-pools.yaml two kafkanodepools of type broker are included, shouldn't they be two of type broker and controller respectively?

### Expected behavior

apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaNodePool
metadata:
  name: pool-b
  labels:
    strimzi.io/cluster: my-cluster
spec:
  replicas: 3
  roles:
    - broker <-- ¿should be controller in pool-b and broker in pool-a?
  storage:
    type: jbod
    volumes:
      - id: 0
        type: persistent-claim
        size: 200Gi
        deleteClaim: false

### Strimzi version

0.40.0

### Kubernetes version

kubenetes v1.28.7

**Comment by ppatierno** [comment]
That example is not about a KRaft cluster. It's about a ZooKeeper based cluster where you deploy 6 brokers in two different node pools. The KRaft related examples are under the examples/kafka/kraft folder instead.
